CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Changelog
 
 
-## 0.55.1
+## 0.56.0
```

Comment:
I agree with that but I think everything depends on the nature of this issue. If I recall correctly, the column `target_column` was introduced by me and wasn't following the convention of using `the_geom` so I made a mistake so we are fixing this.